geth/api/backend.go

```diff
@@ -10,20 +10,26 @@ import (
 	"github.com/status-im/status-go/geth/jail"
 	"github.com/status-im/status-go/geth/log"
 	"github.com/status-im/status-go/geth/node"
+	"github.com/status-im/status-go/geth/notification/fcm"
 	"github.com/status-im/status-go/geth/params"
 	"github.com/status-im/status-go/geth/signal"
 	"github.com/status-im/status-go/geth/txqueue"
 )
 
+const (
+	//todo(jeka): should be removed
+	fcmServerKey = "AAAAxwa-r08:APA91bFtMIToDVKGAmVCm76iEXtA4dn9MPvLdYKIZqAlNpLJbd12EgdBI9DSDSXKdqvIAgLodepmRhGVaWvhxnXJzVpE6MoIRuKedDV3kfHSVBhWFqsyoLTwXY4xeufL9Sdzb581U-lx"
+)
+
 // StatusBackend implements Status.im service
 type StatusBackend struct {
 	sync.Mutex
-	nodeReady      chan struct{} // channel to wait for when node is fully ready
-	nodeManager    common.NodeManager
-	accountManager common.AccountManager
-	txQueueManager common.TxQueueManager
-	jailManager    common.JailManager
-	// TODO(oskarth): notifer here
+	nodeReady       chan struct{} // channel to wait for when node is fully ready
+	nodeManager     common.NodeManager
+	accountManager  common.AccountManager
+	txQueueManager  common.TxQueueManager
+	jailManager     common.JailManager
+	newNotification common.NotificationConstructor
```

Comment:
I still have no idea why we couldn't stay with `common.NotificationManager`. We are introducing a new keyword `Constructor` and it's gonna be hard to explain it to newcomers. 

A call:
```go
err := api.b.newNotification().Send(msg, token)
```
would be
```go
err := api.b.notificationManager.GetClient().Send(msg, token)
```

Reply:
We need to get new one notification for each request or it'll be data race. This is the first reason. The second one, actually i think
`err := api.b.newNotification().Send(msg, token)`
is much cleaner: it doesnt expose internal interfaces or parts like `.GetClient`.